teuthology/task/install.py

```diff
@@ -336,7 +335,8 @@ def _update_deb_package_list_and_install(ctx, remote, debs, config):
     remote.run(args=['sudo', 'apt-get', 'update'], check_status=False)
     remote.run(
         args=[
-            'sudo', 'DEBIAN_FRONTEND=noninteractive', 'apt-get', '-y', '--force-yes',
+            'sudo', 'DEBIAN_FRONTEND=noninteractive', 'apt-get', '-y',
+            '--force-yes',
```

Comment:
You know, if it was Nit-Picking Friday I would say it needs to be a separate commit :D